packages/core/integration-tests/package.json

```diff
@@ -36,8 +36,8 @@
     "parcel-bundler": "2.0.0-beta.1",
     "parcel": "2.0.0-beta.1",
     "postcss-custom-properties": "^8.0.9",
-    "postcss-import": "^12.0.1",
-    "postcss": "^8.0.0",
+    "postcss-import": "^13.0.0",
```

Comment:
One interesting situation that I ran into was that the postcss-import didn't use postcss 8 and inserted nodes into the AST that were missing some methods from recent my PostCSS PR (`toJSON`). This caused a build error